docs/sources/developers/plugins/build-a-streaming-data-source-plugin.md

```diff
@@ -4,29 +4,35 @@ title: Build a streaming data source plugin
 
 # Build a streaming data source plugin
```

Comment:
should this be a sub page of Create a Grafana plugin?

Reply:
Yes, it will move there in phase 2.